recipe/meta.yaml

```diff
@@ -26,7 +26,7 @@ outputs:
         - numpy >=1.20.0
         - pandas >=1.2.5
         - scipy >=1.3.3
-        - scikit-learn >=0.24.0
+        - scikit-learn >=0.24.0,<1.0
```

Comment:
Manually changing these to the correct version to get the CI to pass. The scikit-learn version is correct but the sktime version is not.

When we go to do the next conda release, the sktime version will be the correct one in `latest_release_changes` otherwise the CI will fail.